Source/Core/DolphinQt2/QtUtils/QInputDialog.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/QInputDialog.h"
+
+#include "Core/HotkeyManager.h"
+
+QInputDialog::QInputDialog(QWidget* parent) : QDialog(parent)
```

Comment:
Don't prefix classes with `Q`, it clashes with Qt: [http://doc.qt.io/qt-5/qinputdialog.html](http://doc.qt.io/qt-5/qinputdialog.html). Also, this class name is not descriptive of what it does or why you'd use it.

Besides that, there's a bigger issue here in that now individual dialogs are responsible for knowing whether or not they allow hotkeys. A cleaner way to do this is in the parent class with event filters and signals (#5504 has an example, `DoubleClickEventFilter`).

Stepping back a sec, I know that the behavior here is how DolphinWX does it, but is there a reason hotkeys need to be enabled outside of the render window? @MayImilae